src/common/modal/RefreshModal.vue

```diff
@@ -177,6 +179,12 @@ const isSubmoduleStepRefreshCanBeShown = computed(
     Boolean(submodules.value.length),
 )
 
+const submoduleName = computed(() =>
```

Comment:
I think we can use destructuring
`const submoduleName = computed(() => {
  const { value } = submoduleToSearch;
  return value.charAt(0) === '/' ? value.slice(1) : value;
});`